lib/rubygems/specification.rb

```diff
@@ -2718,6 +2718,8 @@ def reset_nil_attributes_to_default
     end
 
     @installed_by_version ||= nil
+
+    nil
```

Comment:
`@installed_by_version` is just "initialized" in here, return value of this method is never used.